composer.json

```diff
@@ -1,10 +1,22 @@
 {
     "name": "omise/omise-magento",
-    "version": "2.3.0-dev",
+    "description": "Accept payments on your Magento 2 website with Omise",
+    "keywords": ["omise", "payment", "debitcard", "creditcard", "magento"],
```

Comment:
@jonrandy nice for this! can you also add `internetbanking`? We also support that one.

Reply:
ok